client/src/main/java/application/Launcher.java

```diff
@@ -36,7 +36,8 @@ public void start(Stage stage) throws Exception {
       * at the start of the program. For example to skip the login stage and test your own
       * window. */
     ViewFactory viewFactory = new ViewFactory(mainConnection);
-    viewFactory.showStreamWindow(stage);
```

Comment:
Can this be set back to the StreamWindow just to avoid other pull requests doing it after this has been merged?

Reply:
Yeah that seems sensible - I think that because of the way commits work this would be likely not be overwritten by other merges and would stay in the dev branch until manually changed (no-one will have a commit that actively makes that line still be streamWindow)